### PR TITLE
Update Jiminy doc.txt

### PR DESCRIPTION
**Pull request by Josar**
Added that a brown-out happens when 1.8V is set at the DC/DC.
Added explanation for fuse settings.

### CODE REVIEW

File: boards/jiminy-mega256rfr2/doc.txt [hunk 1]
```diff
@@ -133,4 +133,23 @@ This should take care of everything!
 
 RIOT's Makefile are configured to flash the jiminy using AVRDUDE. The bootloader automatically matches
 to the configured baud rate which is set for AVRDUDE. Rates of up to 500kBaud can be used.
- */
\ No newline at end of file
+
+# Special Notes
```
Comment (PeterKietzmann) [inline boards/jiminy-mega256rfr2/doc.txt]: How about "troubleshooting"?

File: boards/jiminy-mega256rfr2/doc.txt [hunk 2]
```diff
@@ -133,4 +133,23 @@ This should take care of everything!
...
+
+Setting the TPS6274x output voltage to 1.8V leads to a undervoltage and thus triggers the brown-out reset.
```
Comment (PeterKietzmann) [inline boards/jiminy-mega256rfr2/doc.txt]: a**n** undervoltage

File: boards/jiminy-mega256rfr2/doc.txt [hunk 3]
```diff
@@ -133,4 +133,23 @@ This should take care of everything!
...
+Disabling the brown-ouit detection with the fuses is recommended when a 1.8V supply is used.
```
Comment (PeterKietzmann) [inline boards/jiminy-mega256rfr2/doc.txt]: ouit->out

File: boards/jiminy-mega256rfr2/doc.txt [hunk 4]
```diff
@@ -133,4 +133,23 @@ This should take care of everything!
...
+
+Reading out the fuses can be done with 
+`avrdude -c wiring -p m256rfr2 -P /dev/ttyACM0 -b 0010005 -U lfuse:r:-:i -v`
+
+The last line should read 
+`avrdude: safemode: Fuses OK (E:FE, H:D0, L:E2)`
+
+extended fuse = FE
```
Comment (PeterKietzmann) [inline boards/jiminy-mega256rfr2/doc.txt]: Capital E?

File: boards/jiminy-mega256rfr2/doc.txt [hunk 5]
```diff
@@ -133,4 +133,23 @@ This should take care of everything!
...
+High fuse     = D0
+Low fuse      = E2
+
+To program this fuse setting execute following line 
+`avrdude -c wiring -p m256rfr2 -P /dev/ttyACM0 -b 0010005 -U lfuse:w:0xe2:m -U hfuse:w:0xd0:m -U efuse:w:0xfe:m`
```
Comment (PeterKietzmann) [inline boards/jiminy-mega256rfr2/doc.txt]: Wouldn't `0xfe` set the `BODLEVEL` bits to `0b110` which means 1,8V brown-out detection? I expected `0xff` (See Table 35-23 in the [reference manual](http://ww1.microchip.com/downloads/en/DeviceDoc/Atmel-8393-MCU_Wireless-ATmega256RFR2-ATmega128RFR2-ATmega64RFR2_Datasheet.pdf)).

**Comment by PeterKietzmann** [comment]
And sorry for all the single comments, this was not intended. Could you also change your commit message to something like:*"boards/jiminy-256rfr2: add information about brown-out fuse setting to doc"

**Comment by Josar** [comment]
@PeterKietzmann added more information and change the commit message, but had to force update for that.

File: boards/jiminy-mega256rfr2/doc.txt [hunk 6]
```diff
@@ -133,4 +133,36 @@ This should take care of everything!
...
 RIOT's Makefile are configured to flash the jiminy using AVRDUDE. The bootloader automatically matches
 to the configured baud rate which is set for AVRDUDE. Rates of up to 500kBaud can be used.
- */
\ No newline at end of file
+
+# Fuse Settings
+Reading out the fuses can be done with
+`avrdude -c wiring -p m256rfr2 -P /dev/ttyACM0 -b 0010005 -U lfuse:r:-:i -v`
+
+The last line should read
```
Comment (PeterKietzmann) [inline boards/jiminy-mega256rfr2/doc.txt]: Remove trailing whitespace

File: boards/jiminy-mega256rfr2/doc.txt [hunk 7]
```diff
@@ -133,4 +133,36 @@ This should take care of everything!
...
+
+Setting the TPS6274x output voltage to 1.8V may lead to an undervoltage and thus triggers the brown-out reset.
+Disabling the brown-out detection by setting the extended fuse bit is recommended when a 1.8V supply is used.
+
+`avrdude -c wiring -p m256rfr2 -P /dev/ttyACM0 -b 0010005 -U efuse:w:0xff:m`
+
+## Using the external crystal oscillator (Transceiver Crystal Oscillator) and deep sleep
+
+When the external crystal oscillator is used as system clock and the device is put into deep sleep mode it seems that
+the clocks for all peripherals are enabled and set to the smallest devider (highest frequency). This leads to a higher
```
Comment (PeterKietzmann) [inline boards/jiminy-mega256rfr2/doc.txt]: devider-> divider

File: boards/jiminy-mega256rfr2/doc.txt (hunk 5 above)
Comment (Josar) [reply boards/jiminy-mega256rfr2/doc.txt]: could you use the `-v ` option and post the whole output! 

Seems to be a usb atmega16 communication issue.

**Comment by Josar** [comment]
@PeterKietzmann squashed.

File: boards/jiminy-mega256rfr2/doc.txt (hunk 7 above)
Comment (PeterKietzmann) [reply boards/jiminy-mega256rfr2/doc.txt]: Not fixed yet. Is it?

**Comment by PeterKietzmann** [comment]
> Wouldn't `0xfe` set the `BODLEVEL` bits to `0b110` which means 1,8V brown-out detection? I expected `0xff` (See Table 35-23 in the [reference manual](http://ww1.microchip.com/downloads/en/DeviceDoc/Atmel-8393-MCU_Wireless-ATmega256RFR2-ATmega128RFR2-ATmega64RFR2_Datasheet.pdf)).

File: boards/jiminy-mega256rfr2/doc.txt (hunk 7 above)
Comment (PeterKietzmann) [reply boards/jiminy-mega256rfr2/doc.txt]: Well, I addressed it and directly pushed to your branch. Guess I should've asked, but it happened more when testing this functionality – sorry for that.

File: boards/jiminy-mega256rfr2/doc.txt (hunk 5 above)
Comment (Josar) [reply boards/jiminy-mega256rfr2/doc.txt]: The bootloader does not allow to flash fusebits. You need to usw an ISP programmer for that.

**Comment by PeterKietzmann** [comment]
@Josar  if an ISP is needed then the doc should state that and explain how to attach it. That's what it's for, right?

> For me the 1,8V works with enabled Brownout detection.

Sure, but the reason we opened this doc PR was that #7943 triggered brown-out detection for some people (me) even with `efuse=0xfe`. So I suggest we leave the example command as is but explain that in certain cases (dc-dc converter @ 1,8V) one might want(?!) to fully disable it with `efuse=0xff`

**Comment by Josar** [comment]
@PeterKietzmann tried to cover all your remarks.

**Comment by PeterKietzmann** [comment]
Please squash

**Comment by Josar** [comment]
@PeterKietzmann squashed and rebased.

**Comment by miri64** [comment]
There are a lot of `doc.txt` within the repo, please use a more specific title.

**Comment by PeterKietzmann** [comment]
@miri64 this is not a new file **and** this is the regular board documentation file. All board docs are named that way.

**Pull request merged**